components/mosquitto/port/include/mosq_broker.h

```diff
@@ -4,6 +4,9 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 #pragma once
+#ifdef __cplusplus
+extern "C" {
+#endif
 #include "mosquitto.h"
 #include "esp_tls.h"
```

Comment:
Nitpick: please place the C++ sentinels below the includes.

```suggestion
#include "mosquitto.h"
#include "esp_tls.h"

#ifdef __cplusplus
extern "C" {
#endif
```